logos/logos-orchestrator/tests/unit/capacity/test_capacity_planner_kv_envelope.py

```
@@ -224,3 +224,114 @@ def test_select_kv_pair_only_uses_fitting_pairs():
+    """A CapacityPlanner stub exposing just what _estimate_available_for_kv_mb needs."""
+    from unittest.mock import MagicMock
+
+    planner = CapacityPlanner.__new__(CapacityPlanner)
+    ledger = MagicMock()
+    ledger.get_effective_available_mb.side_effect = lambda _pid, raw: raw
+    planner._vram_ledger = ledger
+    return planner
+
+
+def test_estimate_available_for_kv_uses_node_ownership_basis_not_transient_free():
+    """KV is sized against the room the lane OWNS post-placement (total VRAM −
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)